### PR TITLE
fix: remove side effect of importing s3_bucket

### PR DESCRIPTION
**Pull request by Bento007**
### Reviewers
**Functional:** @danieljhegeman 

**Readability:** @ebezzi  

---

## Changes
- fixes cxg_admin when `DEPLOYMENT_STAGE` is not set.

### CODE REVIEW

File: scripts/cxg_admin.py [hunk 1]
```diff
@@ -320,7 +320,7 @@ def create_cxg_artifacts(ctx):
         for dataset in datasets:
             if dataset.explorer_url:
                 object_key = dataset.explorer_url.split("/")[-2]
-                s3_uri = f"s3://{cxg_bucket.name}/{object_key}/"
+                s3_uri = f"s3://{S3.cxg_bucket.name}/{object_key}/"
```
Comment (danieljhegeman) [inline scripts/cxg_admin.py:323]: where is `S3` coming from?

File: tests/unit/backend/corpora/common/entities/test_dataset_asset.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def test__generate_file_url__OK(self):
         self.assertIn(file_name, url)
         self.assertIn("Expires=", url)
 
-    @mock.patch("backend.corpora.common.entities.dataset_asset.s3_client")
+    @mock.patch("backend.corpora.common.entities.dataset_asset.s3.client")
```
Comment (danieljhegeman) [inline tests/unit/backend/corpora/common/entities/test_dataset_asset.py:42]: I must really have more python to learn--where is this coming from? I see no `s3` attribute/method/class in `DatasetAsset` or `Entity` 😅

Comment (danieljhegeman) [reply tests/unit/backend/corpora/common/entities/test_dataset_asset.py:42]: ```make local-unit-test-backend path='tests.unit.backend.corpora.common.entities.test_dataset_asset.test__generate_file_url__ERROR'```

gives 

```AttributeError: module 'tests.unit.backend.corpora.common.entities.test_dataset_asset' has no attribute 'test__generate_file_url__ERROR'```

File: scripts/cxg_admin.py (hunk 1 above)
Comment (Bento007) [reply scripts/cxg_admin.py:323]: that's a refactor error. Good catch.

**Review by danieljhegeman — APPROVED** [review]
Looks good. 👍

**Pull request merged**